src/parser/smt2/Smt2.g

```diff
@@ -728,10 +728,9 @@ setInfoInternal[std::unique_ptr<cvc5::Command>* cmd]
   std::string name;
   api::Term sexpr;
 }
-  : KEYWORD symbolicExpr[sexpr]
-    { name = AntlrInput::tokenText($KEYWORD);
-      cmd->reset(new SetInfoCommand(name.c_str() + 1, sexprToString(sexpr)));
-    }
+  : KEYWORD { name = AntlrInput::tokenText($KEYWORD); }
+    symbolicExpr[sexpr]
+    { cmd->reset(new SetInfoCommand(name.c_str() + 1, sexprToString(sexpr))); }
```

Comment:
It is not quite clear to me why this change is needed. Could you explain?

Reply:
I am not too sure either. But it looks like the value stored in `KEYWORD` gets overridden somewhere in `symbolicExpr[sexpr]`, hence, the missing `source` attribute in the printed command. Andy suggested that I immediately store its value `name` and that seems to solve the problem.